Keep a Changelog (#467)
open-telemetry/opentelemetry-dotnet-instrumentation: docs/config.md

| Environment variable | Description | Default |
|-|-|-|

> | `OTEL_TRACES_EXPORTER` | The traces exporter to be used. Available values are: `zipkin`, `jaeger`, `otlp`, `none`. | `otlp` |

Reviewer: Can more than one exporter be defined? If that's the case, are values comma-separated, or it's a list object?

Author: No.  Only one value can be provided.
Maybe 

```suggestion
| `OTEL_TRACES_EXPORTER` | The traces exporter to be used. The value can be one of: `zipkin`, `jaeger`, `otlp`, `none`. | `otlp` |
```
